hr_employee_identification/views/hr_employee_identification_type.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="hr_employee_identification_type_view_form" model="ir.ui.view">
+            <field name="model">hr.employee.identification.type</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
+                        <group>
+                           <field name="name" />
+                       </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+        <record id="hr_employee_identification_type_view_search" model="ir.ui.view">
+            <field name="model">hr.employee.identification.type</field>
+            <field name="arch" type="xml">
+                <search>
+                    <field name="name" />
+                </search>
+            </field>
+        </record>
+        <record id="hr_employee_identification_type_view_tree" model="ir.ui.view">
+            <field name="model">hr.employee.identification.type</field>
+            <field name="arch" type="xml">
+                <tree>
```

Comment:
I think it's better if we use only tree view with `editable="top"`/`editable="bottom"`. Because that model has only 1 field! :D